README.md

```diff
@@ -51,7 +57,25 @@ List of plugins:
 
 
 ## Quickstart
-To run Tempest, please follow the steps in the [Tempest QuickStart] documentation.
+Running the tempest smoke suite against a deployed OpenStack cloud would look like this.
+
+```
+# Install the tempest snap from the latest/edge channel
+sudo snap install tempest --edge
+
+# Have the cloud credentials in the environment
+source novarc
+
+# Initialize a tempest workspace directory called cloud-01
+tempest init cloud-01 && cd cloud-01
+
+# Generate the tempest configuration specific to the cloud
+discover-tempest-config
```

Comment:
We should wait for this default alias to take effect before merging the PR.

Reply:
it took effect. resolving this conversation.

Reply:
Adding link to the [request](https://forum.snapcraft.io/t/auto-connect-alias-request-for-tempest/36422).